### PR TITLE
Use new display data structure

### PR DESCRIPTION
**Pull request by settinghead**
- Error-message component

### CODE REVIEW

File: src/components/financialLists/financialLists.controller.js [hunk 1]
```diff
@@ -45,8 +53,24 @@ class FinancialListsController {
...
+          ctrl.displayInfo = await displayValidationService.validateAndGet( displayId );
+          ctrl.displayId = displayId;
+        } catch ( e ) {
+          _outputErr( "Failed to validate display", e );
+        }
+      } );
+
+    function _outputErr( msg, e ) {
+      console.error( e );
+      alert( `${e.message}` );
```
Comment (tejohnso) [inline src/components/financialLists/financialLists.controller.js]: Does this one have to be an alert?

File: src/components/financialLists/financialLists.controller.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 class FinancialListsController {
-  constructor( financialListAddService, financialListRemoveService, $state, authService, displayUsersService, $async ) {
+  constructor( financialListAddService, $state,
+    authService, displayValidationService, $async,
+  financialListRemoveService, displayUsersService ) {
     "ngInject";
 
+    const ctrl = this;
```
Comment (tejohnso) [inline src/components/financialLists/financialLists.controller.js]: It was working without this extra variable before.  Why not just use *this* along with arrow functions?

File: src/components/financialLists/financialLists.controller.js (hunk 1 above)
Comment (settinghead) [inline src/components/financialLists/financialLists.controller.js]: @tejohnso no. I missed this one. Will change.

File: src/components/financialLists/financialLists.controller.js (hunk 2 above)
Comment (settinghead) [inline src/components/financialLists/financialLists.controller.js]: @tejohnso fixed.

**Comment by settinghead** [comment]
@tejohnso all fixed. Thanks.

**Pull request merged**